alerts/openshift-container-storage-operator/StorageAutoScalingFailed.md

```diff
@@ -0,0 +1,38 @@
+# StorageAutoScalingFailed
+
+## Meaning
+
+When scaling the system to accomodate additional storage,failures may occue due to various factors.In such instances, the `StorageAutoScalingFailed` alert is fired.
+
+## Impact
+
+The storage cannot be scaled at the moment.
+
+## Diagnosis
+
+There can be multiple reasons why the alert has been fired:
+
+- **AWS unable to match storage:**
+   In such scenarios, the first step is to detect the poubt if fauilure by examining the Persistent Volume(PV) and the Persistent Volume Claim (PVC) states. Check for any PVCs in `Pending` or `Bound` state using:
+   ```bash
+   oc describe pvc PVC_NAME
+   ```
+
+   Check that the required permissions and volume types have been set correctly.
+   Additionally, list all PVs with `oc get pv` and verify that the pvcs are correctly recognised and mapped to the osds.
+
+- **OSD is not starting:**
+   ```bash
+   oc get pods -n <namespace>
+   ```
+
+   If the osd is stuck in "starting" stage, high chance the cluster resources might be exhausted. Check for the description of the osd pod.
+   ```bash
+   oc describe pod OSD_POD
+   ```
+
+## Mitigation
+
+- If AWS is unable to allocate storage, verify with the storage provider whether the utilized resources have reached their capacity limits.
+
+- Check upon the Ceph Storage: In case of issues with new OSDs, restart them and try again.
```

Comment:
also try to add try restarting rook operator,
also provide a cmd how to res-start rook operator